README.md

```diff
@@ -68,6 +68,16 @@ The library provides:
 * descriptive and detailed error messages
 * ...and more (over 175 different API calls)
 
+As libgit2 is in no way affiliated with the official Git project, we have to
```

Comment:
This bit seems harsh. How about "As libgit2 is a purely a consumer of the Git system, we have to..."